service/resource/persistentvolume/resource.go

```diff
@@ -159,14 +159,18 @@ func (r *Resource) newRecycleStateAnnotation(pv *apiv1.PersistentVolume, recycle
 // which bounds persistent volume from function parameter.
 func newPvc(pv *apiv1.PersistentVolume) *apiv1.PersistentVolumeClaim {
 
+	storageClassAnnotationValue := getVolumeAnnotation(pv, storageClassAnnotation)
+
 	pvc := &apiv1.PersistentVolumeClaim{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      fmt.Sprintf("pv-cleaner-claim-%s", pv.Name),
 			Namespace: "kube-system",
+			Annotations: map[string]string{
+				"volume.beta.kubernetes.io/storage-class": storageClassAnnotationValue,
```

Comment:
Why not using here `storageClassAnnotation` constant instead of the string literal?

Reply:
nice catch!